app.getclipboard.Clipboard.yml

```diff
@@ -0,0 +1,33 @@
+app-id: app.getclipboard.Clipboard
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+command: cb
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --filesystem=host
+  - --filesystem=xdg-run/Clipboard
+modules:
+  - name: cb
+    buildsystem: cmake-ninja
+    build-commands:
+      - install -D app.getclipboard.Clipboard.metainfo.xml /app/share/metainfo/app.getclipboard.Clipboard.metainfo.xml
+      - install -D app.getclipboard.Clipboard.desktop /app/share/applications/app.getclipboard.Clipboard.desktop
+      - install -D app.getclipboard.Clipboard.png /app/share/icons/hicolor/512x512/apps/app.getclipboard.Clipboard.png
+      - install -D app.getclipboard.Clipboard-128.png /app/share/app-info/icons/flatpak/128x128/app.getclipboard.Clipboard.png
```

Comment:
```suggestion
```

this is not where you install icons. It is plain wrong.

Reply:
where do I install them? this is where jellyfin installs it too: https://github.com/flathub/org.jellyfin.JellyfinServer/blob/a14bdd8f4a799a7459a0a9074f02822c5ba7ae61/org.jellyfin.JellyfinServer.yml#L700

Reply:
where should the icon go?

Reply:
I will reiterate: REMOVE THIS AS REQUESTED